templates/zerver/register.html

```diff
@@ -40,8 +40,7 @@
             {% else %}
                 <input id="id_full_name" class="required" type="text" name="full_name"
                 value="{% if full_name %}{{ full_name }}{% elif form.full_name.value() %}{{ form.full_name.value() }}{% endif %}"
-                maxlength={{ MAX_NAME_LENGTH }} required />
-                <label for="id_full_name" class="inline-block label-title">{{ _('Full name') }}</label>
+                maxlength={{ MAX_NAME_LENGTH }} placeholder="{% trans %}Full name{% endtrans %} or 名前" required />
```

Comment:
The translation here is a bit tricky, since "or" also needs to be translated, and for the japanese translation we wouldn't want "名前 or 名前". Maybe put the whole thing into the `{% trans %}`, and the translators can figure out what to do from context?